admin/app/AppLoader.scala

```diff
@@ -39,15 +39,15 @@ trait AdminServices extends I18nComponents {
   def pekkoActorSystem: PekkoActorSystem
   implicit val executionContext: ExecutionContext
   lazy val capiHttpClient: HttpClient = wire[CapiHttpClient]
-  lazy val contentApiClient = wire[ContentApiClient]
-  lazy val ophanApi = wire[OphanApi]
-  lazy val emailService = wire[EmailService]
-  lazy val fastlyStatisticService = wire[FastlyStatisticService]
-  lazy val fastlyCloudwatchLoadJob = wire[FastlyCloudwatchLoadJob]
-  lazy val redirects = wire[RedirectService]
-  lazy val r2PagePressJob = wire[R2PagePressJob]
-  lazy val analyticsSanityCheckJob = wire[AnalyticsSanityCheckJob]
-  lazy val rebuildIndexJob = wire[RebuildIndexJob]
+  lazy val contentApiClient: ContentApiClient = wire[ContentApiClient]
```

Comment:
I think defining the type explicitly is necessary for `implicits` only.

Reply:
Yep, I closed this one in favour of: 
* https://github.com/guardian/frontend/pull/26806. 

Tried to use the [`scalafix ExplicitResultTypes`](https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes.html) here but didn't notice there was an `onlyImplicits` config.